### ISSUE TITLE
Tab navigation doesn’t work on the second DynamicDialog

### ISSUE THREAD

**Issue by umbarov** [primefaces/primeng#11779]
### Describe the bug

I've have 2 dynamic dialogs. On the first one the tab navigation works, but when I open the second dynamic dialog from the first one, tab navigation stops working on the second dynamic dialog form. The demo:
https://angular-ivy-mpvxm5.stackblitz.io

https://stackblitz.com/edit/angular-ivy-mpvxm5?file=src/app/components/users/user-edit/user-edit.component.html

### Environment

os: macOS monterey, ver 12.5
node: v16.14.2
npm: 8.5.0

### Reproducer

https://angular-ivy-mpvxm5.stackblitz.io

### Angular version

13.0.0 and 14.0.0

### PrimeNG version

13.0.2

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

16.14.2

### Browser(s)

Chrome 103

### Steps to reproduce the behavior

1) Open the demo https://angular-ivy-mpvxm5.stackblitz.io
2) Click `Show users modal` to open first dynamic dialog. Here tab navigation works.
3) Click `Show user edit form`. Opens simple form with 3 text inputs, and here tab navigation doesn't work between form inputs.

### Expected behavior
On the second dynamic dialog tab navigation should work between form inputs.

_No response_

**Comment by cetincakiroglu** [comment]
Hi,

Accessibility enhancements are part of feature hours. If desired, we will be quoting you for an estimated time for this particular enhancement.
As a result, we will be giving you a feature hours proposal. We will be quoting you for an estimated time.
